sample-code/src/server.ts

```diff
@@ -62,6 +63,32 @@ app.get('/ai-api/get-deployments', async (req, res) => {
   }
 });
 
+app.get('/langchain/chat', async (req, res) => {
+  try {
+    res.send(await generate());
+  } catch (error: any) {
+    console.error(error);
+    res
+      .status(500)
+      .send('Yikes, vibes are off apparently 😬 -> ' + error.message);
+  }
+});
+
+app.get('/langchain/embedding', async (req, res) => {
+  try {
+    const result = await embedQuery();
+    if (result.length === 0) {
```

Comment:
[req] style guide!
```suggestion
    if (!result.length) {
```

Reply:
While I agree I simply aligned my "new" endpoints with the existing implementation, if I change it here, I assume I should also change the current /embedding endpoint?